praw/models/reddit/live.py

```diff
@@ -570,6 +597,62 @@ def update(
         self.thread._reset_attributes(*data.keys())
 
 
+class LiveThreadStream:
+    """Provides a :class:`.LiveThread` stream.
+
+    Usually used via:
+
+    .. code-block:: python
+
+       for live_update in reddit.live("ta535s1hq2je").stream.updates():
+           print(live_update.body)
```

Comment:
Could you make this indented 4 spaces instead of 3? I plan on standardizing this at some point. 
```suggestion
        for live_update in reddit.live("ta535s1hq2je").stream.updates():
            print(live_update.body)
```